tests/unit_tests/simpll/CMakeLists.txt

```diff
@@ -34,7 +34,7 @@ exec_program(llvm-config ARGS --libs irreader passes support OUTPUT_VARIABLE llv
 exec_program(llvm-config ARGS --system-libs OUTPUT_VARIABLE system_libs)
 string(STRIP ${system_libs} system_libs)
 
-target_link_libraries(runTests gtest simpll-lib ${llvm_libs})
+target_link_libraries(runTests gtest -lpthread simpll-lib ${llvm_libs})
```

Comment:
Maybe we should rather use `find_package(GTest)` which according to [this discussion](https://discourse.cmake.org/t/findgtest-always-links-threads-threads-to-target-gtest-gtest/8703) automatically links `Threads::Threads`. If that works, it seems like a cleaner solution.

Reply:
You are right, I am going to try it and then update the commit.